docsrc/tutorials/deploy_torch_tensorrt_to_triton.rst

```diff
@@ -0,0 +1,212 @@
+Deploying a Torch-TensorRT model (to Triton)
+============================================
+
+Optimization and deployment go hand in hand in a discussion about Machine 
+Learning infrastructure. For a Torch-TensorRT user, network level optimzation 
```

Comment:
For a Torch-TensorRT user, network level optimization to get the maximum performance would already be an area of expertize. Can we rephrase this ? Not sure what is being implied here